src/ICeWall/README.md

```diff
@@ -0,0 +1,15 @@
+# OpenROAD-ICeWall

```

Comment:
How about removing this file and updating the top level README.md so somebody would actually know that this code even exists? I don't see any point in documenting its use outside of ORD when it is inside ORD.

Reply:
Done